app/scripts/views/sign_up.js

```diff
@@ -26,6 +27,7 @@ define(function (require, exports, module) {
   const SignUpMixin = require('views/mixins/signup-mixin');
   const Template = require('stache!templates/sign_up');
 
+  var device = new Device();
```

Comment:
as @shane-tomlinson mentioned `See this section of code. Ideally we could extract that logic and use it in both places.` We cannot make instances of other views in this view.